devicetypes/smartthings/fidure-thermostat.src/fidure-thermostat.groovy

```diff
@@ -191,160 +183,96 @@ def parse(String description) {
 	def result = []
 
 	if (description?.startsWith("read attr -")) {
-
-		//TODO: Parse RAW strings for multiple attributes
-		def descMap = parseDescriptionAsMap(description)
+		def descMap = zigbee.parseDescriptionAsMap(description)
```

Comment:
These changes are more major than the other DTH exception fixes so I'd suggest creating a separate ticket to help ensure it gets tested by QA.